.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,72 @@
+---
+description: Report a bug.
+labels: [bug]
+name: Bug
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: textarea
+
+    attributes:
+      description: >-
+        Describe the issue in detail. Consider including a minimal working
```

Bugs do not work by definition but can be reproduced:

```suggestion
        Describe the issue in detail. Consider including a minimal reproducible
```